restos.json

```diff
@@ -2,5 +2,6 @@
   "lyon":[
     {"name":"L'Absteme", "url":"http://www.labsteme.fr/en/", "type":"😜"},
     {"name":"Stamtich", "url":"https://www.facebook.com/stamtich/", "type":"Alsacien"}
+    {"name":"Les Raffineuses", "url":"41 Rue Professeur Grignard, 69007 Lyon", "type":"benbon"}
```

Comment:
Hey :wave: @titimoby 
add the address in the name ("name, address") and this in url:
http://www.les-raffineuses.fr/

:octocat: ❤️